File: programmers/lv2_review/lv2_3.py
# 멀쩡한 사각형
def solution(w, h):
    return w * h - (w + h - gcd(w, h))


def gcd(a, b):
    if a < b:
        a, b = b, a

    if b == 0:
        return a
    return gcd(b, a % b)

File: programmers/lv2_review/lv2_3.js
function solution(w, h) {
  const gcd = (a, b) => {
    return b === 0 ? a : gcd(b, a % b);
  };
  return w * h - (w + h - gcd(w, h));
}

console.log(solution(100000000, 99999999));
